packages/beachball/src/changelog/renderChangelog.ts

```diff
@@ -1,12 +1,56 @@
 import { PackageChangelog } from '../types/ChangeLog';
-import { renderPackageChangelog } from './renderPackageChangelog';
+import { renderPackageChangelog, defaultRenderers } from './renderPackageChangelog';
+import { ChangelogOptions, PackageChangelogRenderInfo } from '../types/ChangelogOptions';
 
-export function renderChangelog(previous: string, changelog: PackageChangelog, isGroupedChangelog: boolean): string {
-  const previousLogEntries = previous ? '\n' + previous.substring(previous.indexOf('##')) : '';
+export interface MarkdownChangelogRenderOptions extends Omit<PackageChangelogRenderInfo, 'renderers'> {
+  previousContent: string;
+  changelogOptions: ChangelogOptions;
+}
+
+export function renderChangelog(renderOptions: MarkdownChangelogRenderOptions): string {
+  const {
+    previousJson,
+    previousContent,
+    newVersionChangelog,
+    isGrouped,
+    changelogOptions: { renderPackageChangelog: customRenderPackageChangelog, customRenderers },
+  } = renderOptions;
+
+  const previousLogEntries = previousContent ? previousContent.substring(previousContent.indexOf('##')) : '';
```

Comment:
This seems arbitrary now with a custom renderer - not sure you can count on that? This needs to be fixed before we can merge this I believe.

Reply:
At the moment I think maybe the best is to create a marker with a markdown comment that isn't rendered:
https://stackoverflow.com/questions/4823468/comments-in-markdown

Reply:
Do you specifically want one of those special faux-comment types or is a standard HTML `<!-- comment -->` okay? (I think an HTML comment is fine.)

Reply:
On a related note...oops... https://github.com/microsoft/fluentui/blob/master/packages/keyboard-key/CHANGELOG.md#change-log---fluentuikeyboard-key-1